src/label_approved/cli.py

```diff
@@ -99,9 +99,19 @@ def main() -> None:
 
         p_r_reviews = list(p_r.get_reviews())
 
-        approved_reviews = [review for review in p_r_reviews if review.state == "APPROVED"]
-
-        approval_count = len(approved_reviews)
+        approvals = dict()
+        last_approved_review_date = datetime.min.date()
+        for review in p_r_reviews:
+            if review.state == "APPROVED":
+                approvals[review.user] = review
+                last_approved_review_date = review.submitted_at
```

Comment:
This assumes the order of reviews is chronologically, which held true during my tests.